### ISSUE TITLE
Option to show item description on a chat card

### ISSUE THREAD

**Issue by ABrandau** [javierriveracastro/betteroll-swade#391]
When I use an item with BR, the card shows me which options i can use to attack, cover, edges, etc, but when I make a special ability with certain rules, say a spell, I cant find a way to show the players the description of the item, there is a small icon with the item in the card but that only shows the item to me.

Is there a method to show the description or a feature request would be necessary for this?

**Comment by javierriveracastro** [comment]
The only current way is to use the key used to show system cards.

Any suggestion about how to implement this?.

**Comment by ABrandau** [comment]
Well after doing some double checking, I just noticed that vanilla doesnt display the weapon cards to players to begin with, they only get to see the weapon rolls, your cards are shown but the vanilla one isnt.

If you were to take my suggestion, I would make two changes if possible, first add an option to show the item card and description on the chat, and place that option in the menu (the party where you bind cards to ctrl click, shift click, etc)

And the second, which i dont know if its possible, to allow this functionality to edges and special abilities, though I am thinking this goes out of the scope of the module.

**Comment by javierriveracastro** [comment]
The second one is probably trivial. As those cards right now don't do nothing adding just the description to them should be very easy.

I'm not sure about the first option. I'm thinking of maybe add the description collapsed to all item cads (including skills). But it will take some real space. It was there some time ago, but it was removed to reclaim some space for modifiers.

**Comment by ABrandau** [comment]
Well, as i said before, maybe just add another card type that shows the item description, relevant data and nothing else, and allow it to be binded here in the options, then its up to the user if he uses it or not.

![image](https://user-images.githubusercontent.com/11763228/152344360-683cdf50-ae43-44c4-a675-edddd48c776a.png)

**Comment by SalieriC** [comment]
What I could envision would be making the items _name_ at the top of the card clickable to expand the items description. This would take no space at all as long as it isn't expanded. Maybe combined with a setting whether or not non-owners can to this.
Personally I am okay with the way it currently handles it however.

**Comment by javierriveracastro** [comment]
Blocked by #417

**Comment by javierriveracastro** [comment]
I finally added a collapsed description field to item cards when there is a description available.
